src/robothub/replay_camera.py

```diff
@@ -0,0 +1,542 @@
+
+    def setPreviewSize(self, width: int, height: int) -> None:
+        self._preview_width = width
+        self._preview_height = height
+        self._nodes[self.PREVIEW_STREAM_NAME].setMaxDataSize(width * height * 3)
+
+    # @overload
+    # def setPreviewSize(self, width: int, height: int) -> None:
+    #     raise NotImplementedError("This function is not yet implemented")
+    #
+    # @overload
+    # def setPreviewSize(self, size: Tuple[int, int]) -> None:
+    #     raise NotImplementedError("This function is not yet implemented")
+
+    def setRawNumFramesPool(self, arg0: int) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    def setRawOutputPacked(self, packed: bool) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    def setResolution(self, resolution: dai.ColorCameraProperties.SensorResolution) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    def setSensorCrop(self, x: float, y: float) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    def setStillNumFramesPool(self, arg0: int) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    def setStillSize(self, width: int, height: int) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    # @overload
+    # def setStillSize(self, width: int, height: int) -> None:
+    #     raise NotImplementedError("This function is not yet implemented")
+    #
+    # @overload
+    # def setStillSize(self, size: Tuple[int, int]) -> None:
+    #     raise NotImplementedError("This function is not yet implemented")
+
+    def setVideoNumFramesPool(self, arg0: int) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    def setVideoSize(self, width: int, height: int) -> None:
+        self._video_width = width
+        self._video_height = height
+
+    # @overload
+    # def setVideoSize(self, width: int, height: int) -> None:
+    #     raise NotImplementedError("This function is not yet implemented")
+    #
+    # @overload
+    # def setVideoSize(self, size: Tuple[int, int]) -> None:
+    #     raise NotImplementedError("This function is not yet implemented")
+
+    def setWaitForConfigInput(self, wait: bool) -> None:
+        raise NotImplementedError("This function is not yet implemented")
+
+    @property
+    def frameEvent(self) -> dai.Node.Output:
+        raise NotImplementedError("This function is not yet implemented")
+
+    @property
+    def initialControl(self) -> dai.CameraControl:
+        raise NotImplementedError("This function is not yet implemented")
+
+    @property
+    def inputConfig(self) -> dai.Node.Input:
+        return self._input_control
+
+    @property
+    def inputControl(self) -> dai.Node.Input:
+        return self._input_control
+
+    @property
+    def isp(self) -> dai.Node.Output:
```

Comment:
I think it is better for all outputs to do lazy loading. IE:
```
@property
def isp(self) -> dai.Node.Output:
    if self._isp is None:
        self._create_cam_output(pipeline, self.ISP_STREAM_NAME)
    return self._isp

Reply:
1. Lazy loading seems good. 

2. I think the same approach should be used for sending frames to queues (`_self._video_queue.send(...)`), so we can skip computation (`bgr2nv12`, `resize`) if such output is not needed. 

Both 1. and 2. are implemented in new commit.